client/jetpack-cloud/sections/partner-portal/add-stored-credit-card/test/add-stored-credit-card.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import AddStoredCreditCard from '../index';
+
+describe( '<AddStoredCreditCard>', () => {
+	test( 'should render correctly and have an a tag with href', () => {
+		const initialState = {};
+		const mockStore = configureStore();
+		const store = mockStore( initialState );
+
+		render(
+			<Provider store={ store }>
+				<AddStoredCreditCard />
+			</Provider>
+		);
+
+		const aTag = document.body.getElementsByClassName( 'add-stored-credit-card' )[ 0 ];
```

Comment:
Instead of using `document.body` context, you could directly use the container context returned by `render` to make it faster.

```suggestion
		const { container } = render(
			<Provider store={ store }>
				<AddStoredCreditCard />
			</Provider>
		);

		const [ aTag ] = container.getElementsByClassName( 'add-stored-credit-card' );
```

Reply:
@manzoorwanijk I have updated the tests to use containers like your suggestion. Please check - https://github.com/Automattic/wp-calypso/pull/62695/commits/0b95594fd94a95c8a328c63da8d3283810f831b3

Reply:
Yeah, looks good now. Thank you for making those changes.